RELEASE.md

```diff
@@ -1,3 +1,83 @@
+* Add `EXPAND_DIMS` support to NN API delegate TEST:  expand_dims_test
+* `tf.cond` emits a StatelessIf op if the branch functions are stateless and do not touch any resources.
+* Add support of local soft device placement for eager op.
+* Pass partial_pivoting to the `_TridiagonalSolveGrad`.
+* Add HW acceleration support for `LogSoftMax`.
+* Added a function nested_value_rowids for ragged tensors.
+* fixed a bug in histogram_op.cc.
+* Add guard to avoid acceleration of L2 Normalization with input rank != 4
+* Added evaluation script for COCO minival
+* Add delegate support for QUANTIZE
+* tflite object detection script has a debug mode
+* Add `tf.math.cumulative_logsumexp operation`.
+* Add `tf.ragged.stack`.
+* Add delegate support for `QUANTIZED_16BIT_LSTM`.
+* Add an `implementation=3` mode for `tf.keras.layers.LocallyConnected2D` and `tf.keras.layers.LocallyConnected1D` layers using `tf.SparseTensor` to store weights,  allowing a dramatic speedup for large sparse models.
+* `tf.while_loop` emits a StatelessWhile op if the cond and body functions are stateless and do not touch any resources.
+* Refactors code in Quant8 LSTM support to reduce TFLite binary size.
+* Layers now default to float32, and automatically cast their inputs to the layer's dtype. If you had a model that used float64, it will probably silently use float32 in TensorFlow2, and a warning will be issued that starts with Layer "layer-name" is casting an input tensor from dtype float64 to the layer's dtype of float32. To fix, either set the default dtype to float64 with `tf.keras.backend.set_floatx('float64')`, or pass `dtype='float64'` to each of the Layer constructors. See `tf.keras.layers.Layer` for more information.
+* Fix memory allocation problem when calling `AddNewInputConstantTensor`.
+* Delegate application failure leaves interpreter in valid state.
+* Enable the Keras compile API `experimental_run_tf_function` flag by default. This flag enables single training/eval/predict execution path. With this 1. All input types are converted to `Dataset`. 2. When distribution strategy is not specified this goes through the no-op distribution strategy path. 3. Execution is wrapped in tf.function unless `run_eagerly=True` is set in compile.
+* `tf.cond`, `tf.while` and if and while in AutoGraph now accept a nonscalar predicate if has a single element. This does not affec non-V2 control flow.
+* Enables v2 control flow as part of `tf.enable_v2_behavior()` and `TF2_BEHAVIOR=1`.
+* Add check for correct memory alignment to `MemoryAllocation::MemoryAllocation()`.
+* Extracts `NNAPIDelegateKernel` from nnapi_delegate.cc
+* Added support for `FusedBatchNormV3` in converter.
+* A ragged to dense op for directly calculating tensors.
+* Converts hardswish subgraphs into atomic ops.
+* The equality operation on Tensors & Variables now compares on value instead of id(). As a result, both Tensors & Variables are no longer hashable types.
```

Comment:
Move to major features, and rewrite to:

> Adds `enable_tensor_equality()`, which switches the behavior such that 
> 
> * `Tensor`s are no longer hashable
> * `Tensor`s can be compared with `==` and `!=`, yielding a Boolean `Tensor` with element-wise comparison results.
>
> This will be the default behavior in 2.0

Reply:
Done.